airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/typing_deduping/SnowflakeColumnDefinition.java

```diff
@@ -9,4 +9,11 @@
  * destination-snowflake, where we created PK columns as NOT NULL. This caused a lot of problems
  * because many sources emit null PKs. We may want to remove this field eventually.
  */
-public record SnowflakeColumnDefinition(String type, @Deprecated boolean isNullable) {}
+public record SnowflakeColumnDefinition(String type, boolean isNullable) {
```

Comment:
Deprecated on a constuctor was triggering a warning, while the call to isNullable was also triggering a warning on the calling site for deprecation. We're keeping the deprecation, but removing the warning in this class